src/app/zap-templates/zcl/data-model/chip/client-monitoring-cluster.xml

```diff
@@ -18,40 +18,54 @@ limitations under the License.
 <configurator>
     <domain name="CHIP"/>
 
-    <struct name = "MonitoringRegistration" isFabricScoped="true">
+    <struct name = "MonitoringRegistrationStruct" isFabricScoped="true">
         <cluster code="0x1046"/>
```

Comment:
Where did this cluster code come from?  The spec does not have a cluster code assigned yet, which we should fix ASAP.

Reply:
Random one i had chosen when i first did the cluster xml. There is a ticket assigned to @robszewczyk for this.
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/5903